zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -9655,9 +9655,9 @@ components:
       description: Decision instance search filter.
       type: object
       properties:
-        decisionInstanceId:
+        decisionEvaluationInstanceKey:
           type: string
-          description: The ID of the decision instance.
+          description: The key of the evaluated decision instance.
```

Comment:
Here it's called `evaluated decision instance`, let's take this phrase everywhere (I see some places further down this file) 🕵️